app/models/user.rb

```diff
@@ -1,12 +1,22 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,

```

Comment:
- [ ] It is required that the application asks for confirmation of email. In that case, please add devise's `:confirmable` in the user model to enable that feature.